run-system-tests/action.yml

```diff
@@ -118,6 +118,18 @@ runs:
       with:
         go-version: 1.16
 
+    - name: "Download youtube-dl"
+      shell: 'script --return --quiet --command "bash {0}"'
+      run: |
+        sudo curl -L https://yt-dl.org/downloads/latest/youtube-dl -o /usr/local/bin/youtube-dl
```

Comment:
Pipe the output to a build log and only show on error like we do with the CLIs?
echo "Installing youtube-dl"
sudo curl -L https://yt-dl.org/downloads/latest/youtube-dl -o /usr/local/bin/youtube-dl > build.log 2>&1 || { cat build.log && exit 1; }

Reply:
Ah, neat!